qa/rpc-tests/mempool_tx_expiry.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python2
+# Copyright (c) 2018 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#
+# Test proper expiry for transactions >= version 3
+#
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_equal, connect_nodes, \
+    connect_nodes_bi, sync_blocks, start_nodes, sync_blocks, sync_mempools, \
+    wait_and_assert_operationid_status
+
+from decimal import Decimal
+import time
+
+class MempoolTxExpiryTest(BitcoinTestFramework):
+
+    def setup_nodes(self):
+        return start_nodes(4, self.options.tmpdir, [["-nuparams=5ba81b19:200", "-txexpirydelta=4"]] * 4)
+
+    # Test before, at, and after expiry block
+    # TODO: Test case of dependent txs in reorgs
```

Comment:
Is this still TODO?

Reply:
Yes, need to construct dependent transactions